cypress/support/api.commands.js

```diff
@@ -45,6 +45,33 @@ Cypress.Commands.add('userSubscription', () => {
 });
 
 
+//custom command to refresh token -nt
+
+Cypress.Commands.add ('refreshToken', () => { 
+
+  cy.fixture('refresh_token2').then((refresh_token_data) => {
+      var refresh_token2 = refresh_token_data.refresh_token;
+      cy.log(refresh_token2);
+
+      cy.request({
+          method: 'POST',
+          url: Cypress.env('revayz_endpoint') + 'token_generate',
+          body:  {
+            refresh_token: refresh_token2
+          }
+         })   
+         .then ((response) => { 
+          expect(response.body).to.have.property ('idToken');
+          expect(response.status).to.equal (200);
```

Comment:
we don't do assertions in supports. this should be done in test file

Reply:
I moved them